enhancements/compound-dependency-selectors.md

```diff
@@ -170,80 +168,53 @@ against an installable object's properties.
 
 #### Conjunction and disjunction
 
-These compound constraint types are evaluated following their logical definitions.
+These compound constraint types are evaluated following their logical definitions. 
+
+**Note**: Before proceeding, it is important to keep in mind that these constraint are only solvable by 
+a single operator, not multiple. In order to define multiple operators multiple `olm.constraint` 
+values will need to be specified in the properties. 
```

Comment:
> specified in the properties.

Should this be:
```suggestion
values will need to be specified in the dependencies.yaml or properties.yaml. 
```
If so, I recommend only specifying `dependencies.yaml` to direct consumers to the preferred path.